app/models/update_o_auth_access_data_form.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+class UpdateOAuthAccessDataForm
```

Comment:
Can you help me understand why this code has its own model, as opposed to living in the `OAuthAccessData` class, or something else e.g. a service?

Reply:
This article will explain better than me why I use form objects.
But I agree with you it can be confusing to let this code in the models directory.
We could move this code under a `forms` directory if you prefer ?

Reply:
> We could move this code under a forms directory if you prefer?

Sounds good to me!